lib/chef/provider/git.rb

```diff
@@ -355,6 +358,15 @@ def repo_url
           new_resource.repository
         end
       end
+
+      def get_dir(user)
```

Comment:
Can you add yard to this method?

Reply:
Should also really be named something like `get_homedir(user)` since `dir` is ambiguous without reading the actual code.